drivers/hmis/spec/requests/hmis/form_definition_rules_spec.rb

```diff
@@ -11,6 +11,10 @@
 RSpec.describe Hmis::GraphqlController, type: :request do
   include_context 'hmis base setup'
 
+  before(:all) do
+    ::HmisUtil::JsonForms.seed_all
```

Comment:
this started failing, seems like an order-dependent issue

Reply:
Does it make sense to move this into `hmis base setup`?

Reply:
Yes, I think so. I'll give it a try and and see what breaks

Reply:
hmm that looks like a bigger change actually since there's like 90 tests that use that. I might hold off on that for now